src/main/thrift/parquet.thrift

```diff
@@ -663,6 +669,18 @@ struct ColumnChunk {
    * metadata.
    **/
   3: optional ColumnMetaData meta_data
+
+  /** File offset of this column's OffsetIndex **/
```

Comment:
"of this ColumnChunk's" (these are column chunk, not column, specific)

Reply:
Done.